Automattic/delphin
# Fix wrong image paths

This pull request seeks to fix and standardize image urls in order to use one target CDN location, reducing inconsistencies that would lead to upload images to the wrong place. More specifically, it:
- Updates paths to use the production theme
- Updates host names to `s0.wp.com` instead of `s0.wordpress.com` (longer and incorrect)
- Removes urls to the prototype hosted on Heroku
#### Testing instructions
1. Run `git checkout fix/image-paths` and start your server, or open a [live branch](https://delphin.live/?branch=fix/image-paths)
2. Open the [`Home` page](http://delphin.localhost:1337/)
3. Check that everything displays as before
#### Additional notes

There are some associated tasks to take care of:
- [x] Remove the folder at https://s0.wordpress.com/i/delphin
- [x] Add images hosted on Heroku to the production theme
#### Reviews

@Automattic/sdev-feed
